components/sections/Hero.jsx

```diff
@@ -9,7 +9,7 @@ import underlineBottom from "/public/assets/images/underline-bottom.webp";
 const Hero = () => {
     return (
         <section className="flex flex-col pt-9.25 mb-10.75 lg:mb-39.25">
-            <Heading4 className="text-white mb-11.75 font-semibold">Unit A</Heading4>
+            <Heading2 className="text-white mb-11.75 font-semibold">Unit A</Heading2>
```

Comment:
Can we use the `Heading1` instead of the `Heading2` component? The `Unit A` title is supposed to have a size of `59px` not `47px`. 
```suggestion
            <Heading1 className="text-white mb-11.75 font-semibold">Unit A</Heading1>
```